ci/before_install.sh

```diff
@@ -23,20 +23,20 @@ echo 'python info:'
   2>&1 pyenv versions || true
 ) | sed 's/^/  /'
 
+if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
+    echo "Install Python 2."
+    brew install python@2
+    echo "Upgrade Python 3."
+    brew upgrade python
+fi
+
 echo "Upgrade Python 2 pip."
 pip2.7 -q install --user --upgrade pip
 
-if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
-  echo "Install Python 3."
-  brew install python3
-  echo "Upgrade Python 3 pip."
-  pip3 -q install --user --upgrade pip
-else
-  echo "Upgrade Python 3 pip."
-  # Allow failure. pyenv pip3 on travis is broken:
-  # https://github.com/travis-ci/travis-ci/issues/8363
-  pip3 -q install --user --upgrade pip || true
-fi
+echo "Upgrade Python 3 pip."
+# Allow failure. pyenv pip3 on travis is broken:
+# https://github.com/travis-ci/travis-ci/issues/8363
+pip3 -q install --user --upgrade pip || true
```

Comment:
this hack was intentionally linux-only. Has the problem spilled into travis macOS env?

Reply:
No, I'll change this back.